packages/video_player/video_player/example/ios/Runner/Info.plist

```diff
@@ -27,6 +29,10 @@
 		<key>NSAllowsArbitraryLoads</key>
 		<true/>
 	</dict>
+	<key>UIBackgroundModes</key>
+	<array>
+		<string>audio</string>
```

Comment:
If this is necessary to use the feature then the README should be updated with instructions.

Reply:
This is required. audio is a capability that is also linked to `airplay` & `picture in picture`